mail/hooks.py

```diff
@@ -8,6 +8,7 @@
 
 
 website_redirects = [
+	{"source": "/", "target": "/mail"},
```

Comment:
![image](https://github.com/user-attachments/assets/a4672872-f536-4901-b576-565ac1a15c3a)

For logged-in user

![image](https://github.com/user-attachments/assets/8f27a596-f934-4ccf-98b5-cd759c043e0c)

For Guest

IMO, we should only redirect guests from `/` to `/mail` somehow.